interface/forms/fee_sheet/review/js/fee_sheet_core.js

```diff
@@ -117,21 +117,37 @@ function parse_row_justify(row)
     return retval;
 }
 
-function justify_start(evt)
-{
-    var jqElem=$(this);
-    var parent=jqElem.parent()
-    var template_div=parent.find("div.justify_template");
-    if(template_div.length==0)
-    {
-        var template_div=$("<div class='justify_template'></div>");
-        template_div.attr("data-bind","template: {name: 'justify-display', data: justify}");
-        jqElem.after(template_div);
-    }
-    $(".cancel_dialog").click();
-    var current_justify_choices=parse_row_justify(parent.parent());
-    var justify_model=new fee_sheet_justify_view_model(parent.attr("billing_id"),enc,pid,current_justify_choices);
-    ko.applyBindings(justify_model,template_div.get(0));
+function justify_start(evt) {
+    const jqElem = $(this); // make sure is in scope after save.
```

Comment:
thanks @sjpadgett , what does `$(this)` represent here?

Reply:
It's the items clicked element(cpt4). Same as if using event event.currentTarget. Ya want to preserve because other events happen by time done with ajax save

Reply:
I fixed up little dialog and will put up soon...

Reply:
thanks, guess would decipher that from https://github.com/openemr/openemr/blob/f13189161f050a88a625dac9459f96317bbb94ce/interface/forms/fee_sheet/review/js/fee_sheet_core.js#L166

Reply:
this, that, which, where... it's all a mystery to me...:)

Reply:
ok, so this puts a justify_start listener event onto the cpt4 a tag and then sets the title to of that event to be justify_click_title

Reply:
yep and the $(this) in justify_start tells us which procedure item was clicked. Be wary of this in some situations in javascript as this can get lost or change scope on ya. Good practice if using 'this' more than once or late in routine is to copy to a var i.e myThis = this;
It's all a thing..... 
Anyway I put up restyle so our dialog is not so skinny on new feesheets.
All that I did was resize the tables parent container  to give a min-height then revert on cancel or update. Now how clever is that? kidding, a basic fix....

Will bring in unless you want to test